src/modules/dailyCreditsSummary/i18n/fr_FR.ts

```diff
@@ -0,0 +1,102 @@
+export default {
+    amount: 'Quantité',
+    categories: {
+        allianceMission: {
+            regex: /^\[Alliance]/,
+            title: 'Missions d\'Alliance',
+            backgroundColor: '#ff2800',
+            textColor: 'white',
+        },
+        mission: {
+            regex: '^%missions%( \\(Système d\'alarme\\))?( - Fausse alerte)?$',
+            title: 'Missions personnel',
+            backgroundColor: '#ff2400',
+            textColor: 'white',
+        },
+        cancelledMissions: {
+            regex: '^%missions%( \\(Système d\'alarme\\))? - Annulée$',
+            title: 'Missions annulées',
+            backgroundColor: '#7C7978',
+            textColor: 'white',
+        },
+        buyVehicle: {
+            regex: /Véhicule acheté/,
+            backgroundColor: '#007fff',
+            textColor: 'white',
+        },
+        dailyLogin: {
+            regex: /Récompense de connexion quotidienne/,
+            backgroundColor: '#8db600',
+            textColor: 'black',
+        },
+        buildings: {
+            regex: /(Bâtiment  construit|Rembourser le bâtiment$|Building demolished without refund)/,
```

Comment:
double space ` ` in this line by accident

Reply:
thank